ports/esp32/main_esp32/idf_component.yml

```diff
@@ -2,4 +2,4 @@
 dependencies:
   espressif/mdns: "~1.1.0"
   idf:
-    version: ">=5.0.4"
+    version: ">=5.2.0"
```

Comment:
Is this correct even though the IDF tag is `v5.2`?

Reply:
I'm pretty sure it's correct, just wanted to double check.

Reply:
It is, these are SemVer rather than git tags: https://docs.espressif.com/projects/idf-component-manager/en/latest/reference/versioning.html#version-precedence